demo/android/Activity/copy_test_resources.sh

```diff
@@ -1,34 +1,38 @@
-if [ ! -d "./porcupine-activity-demo-app/src/androidTest/assets/test_resources/audio_samples" ]
+LIB_DIR="../../../lib"
```

Comment:
we can also add the contents of this script to the gradle build - though not as important as the binding

Reply:
This one is a bit hard to do in gradle because of the per-language dirs and the re-organization on the destination. I think it's possible but it's a fair amount of work/research for release. Okay if I make a ticket to migrate this one later?